### DISCUSSION TITLE
How to remove MUI Chip in columnFilterModes  for ( empty & not empty) in all the rows.

### DISCUSSION THREAD

**Discussion by piverlc** [KevinVandy/material-react-table#192] [Q&A]
Unable to remove MUI Chip in columFilterModes for (empty & not empty) filter modes. It's internally clearing every columnFilters state but unable to remove above chip component.  

I'm trying to clear every filter using a button, it's working properly but I want to clear the above chip also when i click the button.

![columnFilterModes](https://user-images.githubusercontent.com/91478831/200874744-8acfb81a-190b-437e-a5b7-2abca073a0cd.png)

**Comment by KevinVandy** [comment]
I've had a really similar conversation here https://github.com/KevinVandy/material-react-table/discussions/152

Need to come up with a solution at some point

**Reply by piverlc** [reply]
That would be awesome thankyou.

for now I am resetting the filters like this

`table.setColumnFilterFns({ id: "fuzzy" });`
